src/components/elements/ImageGallery/ImagePreview.tsx

```diff
@@ -0,0 +1,59 @@
+import Link from "next/link";
+import { DetailedHTMLProps, FC, HTMLAttributes } from "react";
+import ReactDOM from "react-dom";
+import { twMerge as tw } from "tailwind-merge";
+
+import IconButton from "@/components/elements/IconButton/IconButton";
+import Icon, { IconNames } from "@/components/extensive/Icon/Icon";
+
+import ImageWithPlaceholder from "../ImageWithPlaceholder/ImageWithPlaceholder";
+import { ImageGalleryItemData } from "./ImageGalleryItem";
+
+export interface ImageGalleryPreviewerProps extends DetailedHTMLProps<HTMLAttributes<HTMLDivElement>, HTMLDivElement> {
+  data: Omit<ImageGalleryItemData, "label" | "value" | "isPublic" | "thumbnailImageUrl"> & { label?: string };
+  onCLose?: () => void;
+  backdropClassName?: string;
+  WrapperClassName?: string;
+}
+
+const ImageGalleryPreviewer: FC<ImageGalleryPreviewerProps> = ({
+  data,
+  onCLose,
+  className,
+  WrapperClassName,
+  backdropClassName,
+  ...rest
+}) => {
+  return ReactDOM.createPortal(
```

Comment:
Please avoid the `return` statement for single statement arrow functions
```
}) => ReactDOM.createPortal(
```

Reply:
done